XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
Fair question! I'm inclined to go all the way and do as the other tools (.gitignore goes to the root of the repo, but stylecop and resharper go all the way to the root)